### DISCUSSION TITLE
🚀 Feature request: Open Structure "View Page" links in new tab

### DISCUSSION THREAD

**Discussion by jaredchesebro** [ExpressionEngine/ExpressionEngine#4117] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
If you click the "View Page" button in Structure, the link opens in the current browser tab. This is not great UX because you have to navigate back to the cp manually in a new browser tab if you then want to make content changes.

![structure-view-page](https://github.com/ExpressionEngine/ExpressionEngine/assets/21958491/cddffc6e-8e27-47d3-ae1a-0e9478c03363)

**Describe the solution you'd like**
Open "View Page" links in a new browser tab instead of the current browser tab. (Be sure make it accessible by letting user know it opens in a new tab).

**Comment by intoeetive** [comment]
This has been requested before in https://github.com/ExpressionEngine/ExpressionEngine/discussions/3834 and there's already a PR addressing it https://github.com/ExpressionEngine/ExpressionEngine/pull/3836
